src/CommunityToolkit.Mvvm/CommunityToolkit.Mvvm.csproj

```diff
@@ -1,7 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netstandard2.1;net6.0;net8.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net6.0;net8.0;net8.0-windows10.0.17763.0</TargetFrameworks>
+    <WindowsSdkPackageVersion>10.0.17763.37-preview</WindowsSdkPackageVersion>
```

Comment:
Repeated twice?  Here and below in condition.

Reply:
Forgot to delete it after I moved them to a property group 😅